openmetadata-ui/src/main/resources/ui/src/components/common/error-with-placeholder/ErrorPlaceHolder.tsx

```diff
@@ -11,31 +11,90 @@
  *  limitations under the License.
  */
 
+import { Typography } from 'antd';
 import React from 'react';
-import NoDataFoundPlaceHolder from '../../../assets/img/no-data-placeholder.png';
+import {
+  default as AddPlaceHolder,
+  default as NoDataFoundPlaceHolder,
+} from '../../../assets/img/no-data-placeholder.svg';
+import { Button } from '../../buttons/Button/Button';
 
 type Props = {
   children?: React.ReactNode;
+  type?: string;
+  buttonLabel?: string;
+  buttonListener?: () => void;
+  heading?: string;
+  doc?: string;
+  buttons?: React.ReactNode;
+  buttonId?: string;
 };
 
-const ErrorPlaceHolder = ({ children }: Props) => (
-  <>
-    <div
-      className="tw-flex tw-flex-col tw-mt-24 tw-place-items-center"
-      data-testid="error">
-      {' '}
-      <img
-        data-testid="no-data-image"
-        src={NoDataFoundPlaceHolder}
-        width="200"
-      />
-    </div>
-    {children && (
+const ErrorPlaceHolder = ({
+  doc,
+  type,
+  children,
+  heading,
+  buttonLabel,
+  buttonListener,
+  buttons,
+  buttonId,
+}: Props) => {
+  const { Paragraph, Link } = Typography;
+
+  return type === 'ADD_DATA' ? (
+    <div>
```

Comment:
Is this empty div required?

Reply:
@Sachin-chaurasiya  It's for the sack of single component return, will use Fragment here instead.